assets/javascripts/discourse/initializers/extend-for-assigns.js

```diff
@@ -556,10 +568,17 @@ function initialize(api) {
           )
         );
       }
+
       if (indirectlyAssignedTo) {
         Object.keys(indirectlyAssignedTo).map((postId) => {
           const assignee = indirectlyAssignedTo[postId].assigned_to;
           const postNumber = indirectlyAssignedTo[postId].post_number;
+
+          displayedName =
+            this.siteSettings.prioritize_full_name_in_ux || !assignee.username
```

Comment:
I don't _think_ we need the `|| !assignee.username`, but I could be wrong!

Reply:
There is an issue when there are groups involved, I thought of leaving a comment in the code. Basically groups doesnt go by username but they use name so that is why I had to add that there

Reply:
Ohhh gotcha, yeah I think a comment would be fine!